core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/CsvRow.java

```diff
@@ -85,6 +85,12 @@ public String asString(int columnIndex) {
         if (columnIndex < 0 || columnIndex >= columnValues.length) {
             return null;
         }
-        return columnValues[columnIndex];
+        String s = columnValues[columnIndex];
+        // Univocity CSV parser already returns null for no explicit value and for an explicit empty string "".
+        // Here we just want to be sure that we always return null and never "".
+        if (s == null || s.isEmpty()) {
```

Comment:
Could Guava [isNullOrEmpty](https://guava.dev/releases/27.1-jre/api/docs/com/google/common/base/Strings.html#isNullOrEmpty-java.lang.String-) be used?

Reply:
Done. Nice!